features/google/src/components/activitiesOverTimeStory/activitiesOverTimeStory.jsx

```diff
@@ -0,0 +1,190 @@
+                    year={selectedYear}
+                    yearRange={yearRange}
+                    onYearChange={setSelectedYear}
+                />
+            ),
+            belowChart: selectedYear,
+        },
+        {
+            id: "yearly",
+            translation: i18n.t("activitiesOverTimeStory:tab.year"),
+            barData: monthlyTotals,
+            barWidth: 22,
+            barChartLegendText: i18n.t(
+                "activitiesOverTimeStory:tab.events.yearly",
+                {
+                    number_events:
+                        activitiesOverTime.values[selectedYear]?.total || 0,
+                }
+            ),
+            barValueColor: "white",
+            datePicker: <div className="datepicker-filler" />,
+            belowChart: i18n.t("common:total.years"),
+        },
+    ];
+
+    //Adapt to order of magnitude
+    const orderOfMagnitude = (data) => {
+        const highest = Math.max(...Object.values(data));
+        if (highest >= 1000) {
+            tabData.barData = Object.fromEntries(
+                Object.entries(data).map(([key, value]) => [key, value / 1000])
+            );
+            return 1000;
+        }
+        return 1;
+    };
+
+    const numberOfEventsString = (data) =>
+        i18n.t(
+            `activitiesOverTimeStory:number.events${
+                orderOfMagnitude(data) > 1 ? ".thousands" : ""
+            }`
+        );
+
+    return (
+        <div className="activities-ministory">
+            <p>
+                {i18n.t("activitiesOverTimeStory:summary", {
+                    number_activities: activitiesOverTime.total,
+                })}
+            </p>
+            <Tabs key={0}>
+                {tabData.map((tab, index) => (
+                    <Tab id={index} key={index} label={tab.translation}>
+                        {tab.datePicker}
+                        <div className="legend">
+                            <div
+                                className="bar-piece"
+                                style={{
+                                    width: tab.barWidth + "px",
+                                }}
+                            ></div>
+                            <p>{tab.barChartLegendText}</p>
+                        </div>
+                        <p className="above-chart">
+                            {numberOfEventsString(tab.barData)}
+                        </p>
+                        <PolyChart
+                            type="vertical-bar-chart"
+                            data={tab.barData}
+                            barColor={"white"}
+                            barWidth={tab.barWidth}
+                            barValueColor={tab.barValueColor}
+                        />
+                        <p className="below-chart">{tab.belowChart}</p>
+                    </Tab>
+                ))}
+            </Tabs>
+            <p className="source">{i18n.t("common:source.your.google.data")}</p>
```

Comment:
There does not seem to be enough differences to actually warrant a separate file... But I guess it's OK as a first version.